st2common/st2common/services/trace.py

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import absolute_import, print_function
```

Comment:
Why only import the print_function here and not anywhere else where there're `print`?

Reply:
fixed in 7bc29d8a0